.github/workflows/release-connect-bump-versions.yml

```diff
@@ -0,0 +1,45 @@
+name: "[Release] connect bump versions"
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        type: choice
+        description: semver
+        options:
+          - patch
+          - minor
+          - prerelease
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  bump-versions:
+    if: github.repository == 'trezor/trezor-suite'
```

Comment:
I noticed that not all the jobs here have this check. should they have it? or is it enough only for the first job to have it?

Reply:
Good point. I actually think that `if: github.repository == 'trezor/trezor-suite'` should be only used in workflows that are run automatically like the tests ones, to avoid running when running in forks. But this one has to be triggered manually so I would say it is not necessarily.  @vdovhanych could you confirm it?

Reply:
Yes, we can get rid of it for release workflows